<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="choosePorts">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="baud_value">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>209700</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="connect_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>200</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open the serial port</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="browser_dataReceive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="edit_dataSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>791</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="box_stopBit">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="box_dataBits">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="box_parityBit">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>170</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Even</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odd</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>550</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clearSend">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>630</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_hexSend">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>590</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX SEND</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ADC_PORT_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ADC_PORT_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_AMP1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_AMP2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_adccalpause">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ADC_Port_1_lb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC Port 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="ADC_Port_2_lb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC Port 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="AMP1_l">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AMP1</string>
    </property>
   </widget>
   <widget class="QLabel" name="AMP2_l">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AMP2</string>
    </property>
   </widget>
   <widget class="QLabel" name="ADCCALPAUSE_l">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AdcCalPause</string>
    </property>
   </widget>
   <widget class="QLabel" name="AutoOffTime">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoOffTime</string>
    </property>
   </widget>
   <widget class="QLabel" name="AdcDiv">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AdcDiv</string>
    </property>
   </widget>
   <widget class="QLabel" name="AdcSmp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AdcSmp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lo_adcckmode">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAutoOffTime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DacDiv">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DacDiv</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_DacDiv">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ADCDiv">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="AdcCkmode_l">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AdcCkmode</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_AdcSmp">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MathAvr">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MathAvr</string>
    </property>
   </widget>
   <widget class="QLabel" name="Kdft">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kdft</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ndft">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ndft</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_TimDiv">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_MathAvr">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="OPAHSM1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OPAHSM1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_OPAHSM1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Kdft">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TimDiv">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TimDiv</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Ndft">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Offse2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset2</string>
    </property>
   </widget>
   <widget class="QLabel" name="OPAHCM3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OPAHSM3</string>
    </property>
   </widget>
   <widget class="QLabel" name="Offset1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_OPAHSM2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_offset2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PGA_CAIN1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PGA_CAIN1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_PGA_CAIN1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_OPAHSM3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="OPAHSM2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OPAHSM2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Offset1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="RefHiz">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RefHiz</string>
    </property>
   </widget>
   <widget class="QLabel" name="PGA_CAIN3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PGA_CAIN3</string>
    </property>
   </widget>
   <widget class="QLabel" name="RefEnvr">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RefEnvr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_PGA_CAIN2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_RefHiz">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="RefVrs">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RefVrs</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_RefVrs">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_PGA_CAIN3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ADC_Port_1_lb_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PGA_CAIN2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_RefEnvr">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ADC_Port_2_lb_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>_DAC1</string>
    </property>
   </widget>
   <widget class="QLabel" name="_DAC2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>_DAC2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Tpp">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_DAC1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Tpp">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tpp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_DAC2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_get">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>GET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_load">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>LOAD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ADC1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="_DAC2_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ADC2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ADC_Port_2_lb_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
